website/docs/docs/collaborate/column-level-lineage.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Column level lineage"
+description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
+---
+
+dbt Explorer now offers column level lineage (CLL) for the resources in your dbt project. Analytics engineers can quickly and easily gain insight into the provenance of their data products at a more granular level. For each column in a dbt resource (model, source, or snapshot), Explorer provides the full lineage for the data in that column.
+
+Column level lineage is available to dbt Cloud Enterprise accounts that can use Explorer. It’s also available through the Discovery API.
+
+:::tip Beta
+Column-level lineage is now available in beta. Check it out! We'd love to [know what you think](https://docs.google.com/forms/d/e/1FAIpQLSdpCbVkGY9QwfExFonpWE4DTOKi3fQxBGLD0wwKYpkMjgcE7g/viewform)!
```

Comment:
i didn't use "open beta" here, aligned with the betas for Project recs and Model perf pages. lemme know if this doesn't work tho! happy to change